site/en/docs/workbox/modules/workbox-background-sync/index.md

```diff
@@ -56,6 +56,7 @@ registerRoute(
 );
 ```
 
+[comment]: <> (TODO: update the using-plugins link when that doc is migrated)
```

Comment:
(I've never seen this syntax for comments in Markdown.)

Reply:
I found it from [this StackOverflow discussion](https://stackoverflow.com/questions/4823468/comments-in-markdown), however, the specification it references doesn't seem to include it anymore. Regardless, it doesn't render as a visible line, so maybe GFM supports it anyway. If you have a preferred syntax, let me know.